### PR TITLE
Upgrade cc/configure_tensorflow.sh

### PR DESCRIPTION
**Pull request by tommadams**
- Upgrade to TF v1.11
- Build the gRPC runtime required by Cloud TPU.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@ autopep8>=1.3
 fire
 google.cloud.logging
 grpcio-tools
+keras
```
Comment (brilee) [inline requirements.txt]: what's this about?

Comment (tommadams) [reply requirements.txt]: We build the tensorflow install package in order to extract the C++ headers and keras is a required TF dependency now :/

I _really_ want to start using the user's TF install.

File: cc/configure_tensorflow.sh [hunk 2]
```diff
@@ -60,29 +58,54 @@ echo "Copying tensor flow headers to ${dst_dir}"
 cp -r ${tmp_dir}/tensorflow-*.data/purelib/tensorflow/include/* "${dst_dir}"
 
 echo "Building tensorflow libraries"
-bazel build -c opt --config=opt --copt="${cc_opt_flags}" //tensorflow:libtensorflow_cc.so //tensorflow:libtensorflow_framework.so
+
+# Add a custom BUILD target for the gRPC runtime.
```
Comment (amj) [inline cc/configure_tensorflow.sh]: this is both hilarious and awful.

Comment (amj) [reply cc/configure_tensorflow.sh]: and i'm also impressed -- i don't think i would've come up with this as a solution XD

**Pull request merged**